src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+      )}
+      <p className="Person__partner">
+        {person.isMarried
+          ? (`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`)
+          : (`I am not married`)}
```

Comment:
Don't use ternary operator inside another ternary operator
In this way it'll be hard to read and understand the code

Will be better to prepare the data before the `return` statement
For example, create variable for `partnerStatus` data (wife or husband) and then use this variable inside of jsx